generators/client/templates/angular/src/main/webapp/app/core/tracker/tracker.service.ts.ejs

```diff
@@ -112,28 +114,29 @@ export class <%=jhiPrefixCapitalized%>TrackerService {
         }
     }
 
-    subscribe() {
-        this.connection.then(() => {
-            this.subscriber = this.stompClient.subscribe('/topic/tracker', (data) => {
-                this.listenerObserver.next(JSON.parse(data.body));
-            });
+    subscribe(): void {
+        if (this.connectionSubscription !== null) {
+            return;
+        }
+      
+        this.connectionSubscription = this.connectionSubject.subscribe(() => {
```

Comment:
Can we avoid nested `subscribe` by replacing the first `subscribe` with `tap` operator  ?

Reply:
These are different subscriptions, outer is `rxjs` one and inner is `Stomp` one, so can't avoid nesting.